com.ibm.wala.core/src/com/ibm/wala/dataflow/IFDS/TabulationSolver.java

```diff
@@ -715,46 +732,46 @@ public void act(final int d1) {
               // was recorded as a summary edge
               IntSet reachedBySummary = summaries.getSummaryEdges(s_p_num, x_num, d1);
               if (reachedBySummary != null) {
-                for (final T returnSite : returnSitesForCallee) {
+                List<T> returnSitesForExit = exitMap.get(exit);
+                if (returnSitesForExit == null) {
+                  return;
```

Comment:
Good catch, this is probably a bug that I introduced (and didn't show up despite running WALA tests). Will investigate and fix.